doc/source/user_guide/parameters.rst

```diff
@@ -42,3 +42,148 @@ ROUT``, you can access it with:
 
 For a full listing of the methods and attributes available to the
 ``Parameters`` class, please reference the :ref:`ref_parameters_api`.
+
+
+Especially Named Parameters
+===========================
+
+Leading Underscored Parameters
+------------------------------
+
+The parameters starting with underscore (``'_'``) are reserved parameters
+for MAPDL macros and routines, therefore its use is discouraged, and
```

Comment:
"routines, their use is discouraged, and in PyMAPDL you cannot set them directly."